test/integration/test-submit-readable-stream.js

```diff
@@ -0,0 +1,58 @@
+var common = require('../common');
+var assert = common.assert;
+var http = require('http');
+var path = require('path');
+var mime = require('mime');
+var request = require('request');
+var fs = require('fs');
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['fs' is defined but never used.](https://www.codacy.com/app/form-data/form-data/file/875565524/issues/source?bid=2805153&fileBranchId=2805172#l7)